jsondiff.go

```diff
@@ -43,34 +43,48 @@ type Tag struct {
 }
 
 type Options struct {
-	Normal     Tag
-	Added      Tag
-	Removed    Tag
-	Changed    Tag
-	Prefix     string
-	Indent     string
-	PrintTypes bool
+	Normal           Tag
+	Added            Tag
+	Removed          Tag
+	Changed          Tag
+	Prefix           string
+	Indent           string
+	PrintTypes       bool
```

Comment:
maybe new indentation is unnecessary, let's wait check what @nsf says

Reply:
Well, it's a result of gofmt, it's fine. Also the whole pull request looks fine too, will merge it in a sec.